tests/system/providers/databricks/example_databricks.py

```diff
@@ -201,6 +202,39 @@
     )
     # [END howto_operator_databricks_notebook_existing_cluster]
 
+    # [START howto_operator_databricks_task_notebook]
+    task_operator_nb_1 = DatabricksTaskOperator(
+        task_id="nb_1",
+        databricks_conn_id="databricks_conn",
+        job_cluster_key="Shared_job_cluster",
+        task_config={
+            "notebook_task": {
+                "notebook_path": "/Shared/Notebook_1",
+                "source": "WORKSPACE",
+            },
+            "libraries": [
+                {"pypi": {"package": "Faker"}},
+                {"pypi": {"package": "simplejson"}},
+            ],
+        },
+    )
+    # [END howto_operator_databricks_task_notebook]
+
+    # [START howto_operator_databricks_task_sql]
+    task_operator_sql_query = DatabricksTaskOperator(
+        task_id="sql_query",
+        databricks_conn_id="databricks_conn",
+        task_config={
+            "sql_task": {
+                "query": {
+                    "query_id": "c9cf6468-babe-41a6-abc3-10ac358c71ee",
```

Comment:
It's an ID from a workspace I'm testing that. This can be treated as an example and users can replace this with their query_id based on their Databricks connection and workspace.

Reply:
Would it be better if we try something like `QUERY_ID = os.environ.get("QUERY_ID", "...")` like other example DAGs?

Reply:
yes, good idea. done so.